nbsite/cmd.py

```diff
@@ -19,6 +20,7 @@
     "FAQ",
     "API"
 ]
+SCRIPT_DIR = str(Path(__file__).parents[1] / "scripts")
```

Comment:
We need to doublecheck that this path is also valid when installing the package.